mlkem/polyvec.c

```diff
@@ -15,7 +15,7 @@ void polyvec_compress_du(uint8_t r[MLKEM_POLYVECCOMPRESSEDBYTES_DU],
                          const polyvec *a)
 {
   unsigned i;
-  debug_assert_bound(a, MLKEM_K * MLKEM_N, 0, MLKEM_Q);
+  debug_assert_bound_2d(a, MLKEM_N, MLKEM_K, 0, MLKEM_Q);
```

Comment:
Can we swap the arguments? `a, K, N` seems more natural to me as we would be writing `a[K][N]` and not a[N][K]`.

Reply:
Ok

Reply:
Changed